cache/models/samples.js

```diff
@@ -145,6 +146,23 @@ function cleanAddAspectToSample(sampleObj, aspectObj) {
   return sampleRes;
 } // cleanAddAspectToSample
 
+/**
+ * Returns true if any attributes are being modified. (Don't bother comparing
+ * the "name" attribute--the only difference would be uppercase/lowercase and
+ * that should not be treated as a change.)
+ *
+ * @param {Object} newSample - received from query body
+ * @param {Object} oldSample - from redis
+ * @returns {Boolean} true if any sample attributes are being modified
+ */
+function isSampleChanged(newSample, oldSample) {
+  const delta = Object.keys(newSample)
+    .filter((key) => key !== 'name')
+    .filter((key) =>
```

Comment:
@iamigo We can also try using just one filter instead of two here - it would still be O(n) - but would not hurt to try.